tests/foreman/maintain/test_backup_restore.py

```diff
@@ -448,7 +448,7 @@ def test_positive_puppet_backup_restore(
     result = sat_maintain.cli.Backup.run_backup(
         backup_dir=subdir,
         backup_type=backup_type,
-        options={'assumeyes': True, 'plaintext': True, 'skip-pulp-content': False},
+        options={'assumeyes': True, 'plaintext': True, 'skip-pulp-content': False, 't': '50M'},
```

Comment:
For better readability 
```suggestion
        options={'assumeyes': True, 'plaintext': True, 'skip-pulp-content': False, 'split-pulp-tar': '50M'},
```